hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/LogEventViewGenerator.java

```diff
@@ -22,6 +26,11 @@
   private static final Logger LOG = LoggerFactory.getLogger(LogEventViewGenerator.class);
   private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
 
+  private static final String LOG_EVENT_ATTRIBUTE_SIZE_METRIC =
+      "hypertrace.log.event.attribute.size";
+  private static final Map<String, AtomicInteger> logEventAttributeSizeGauge =
+      new ConcurrentHashMap<>();
```

Comment:
why is this a gauge ? I think we want to look at the trend of the size ? Seems like it should be a counter ?

Reply:
I basically wanted to report the observed value at the instant https://micrometer.io/docs/concepts#_gauges 
counter would be summing the values reported over the time?

Reply:
Ok. I guess counters are useful if we want rate metrics or averages we can apply rate function in prometheus on counters to get metrics like per size per min etc.